ratpack-bearer-auth/src/main/java/st/ratpack/auth/internal/DefaultOAuthToken.java

```diff
@@ -6,6 +6,7 @@
 
 public class DefaultOAuthToken implements OAuthToken {
 
+	private static final long serialVersionUID = -1236874447952674147L;
 	private final String authToken;
```

Comment:
You have to be careful with using generated serialVUId. This might throw invalidCalssException in certain JVMs. It is better to manage this than using a generated one.

Reply:
I generated it using the [serialver command](https://docs.oracle.com/javase/8/docs/technotes/tools/unix/serialver.html) bundled with the JVM specifically for this purpose. Josh Bloch's _Effective Java_ recommends this approach.

Here is a good article that also recommends this approach: https://www.javaworld.com/article/2072426/a-few-details-about-serialver.html